brume/config.py

```diff
@@ -29,6 +56,16 @@ def is_git_repo():
 
 class Config():
 
+    @staticmethod
+    def cfn(stack_name, key, secondKey=None, thirdKey=None):
```

Comment:
Can you provide a docstring for this method and update the README with an example usage of the `cfn` function in `brume.yaml`?

Also please use snake_case for variables in this method.